src/model/comment/level.rs

```diff
@@ -76,6 +75,67 @@ pub struct LevelComment<'a> {
     pub special_color: Option<Color>,
 }
 
+#[derive(Debug, Eq, PartialEq, Clone, Deserialize, Serialize)]
+pub struct CommentHistory<'a> {
+    /// Information about the user that made this [`LevelComment`]. Is generally a [`CommentUser`]
```

Comment:
drop the "generally"

Reply:
The CommentHistory struct is removed now, I haven't made the commit yet but I will in a bit.